src/google-api.ts

```diff
@@ -0,0 +1,103 @@
+import * as jose from 'jose'
+import type { RespJson } from "./common"
+
+declare global {
+    const cron_worker: KVNamespace;
+}
+
+export class GoogleAPI {
+    authenticator: GoogleJwtAuthenticater;
+    adminScopes = "https://www.googleapis.com/auth/calendar https://www.googleapis.com/auth/admin.directory.group https://www.googleapis.com/auth/admin.directory.group.member https://www.googleapis.com/auth/admin.reports.audit.readonly https://www.googleapis.com/auth/admin.directory.user.security" ;
```

Comment:
Can this be an array? It's easier to read that way.

Reply:
the scope has to be a single string and not an array. Added comments to explain the issue.